Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -964,11 +1020,24 @@ def get_ioc_type(indicator: str, id_to_object: dict[str, dict[str, Any]]) -> str
         indicator_obj = id_to_object.get(indicator, {})
         pattern = indicator_obj.get('pattern', '')
         for stix_type in STIX_2_TYPES_TO_CORTEX_TYPES:
-            if pattern.startswith(f'[{stix_type}'):
+            if pattern.startswith(f'[{stix_type}') and (not STIX2XSOARParser.is_not_supported_iocs_type(pattern)):
                 ioc_type = STIX_2_TYPES_TO_CORTEX_TYPES.get(stix_type)  # type: ignore
                 break
         return ioc_type
 
+    @staticmethod
+    def is_not_supported_iocs_type(pattern: str):
+        """
+        Get pattern and check if the type is not supported by XSOAR.
+
+        Args:
+            pattern: the indicator pattern.
+
+        Returns:
+            bool.
+        """
+        return any(pattern.startswith(f"[{key}") for key in STIX_NOT_SUPPORTED_TYPES)
```

Comment:
```suggestion
        return not any(pattern.startswith(f"[{key}") for key in STIX_NOT_SUPPORTED_TYPES)
```
There is a better way probably haha